src/test/app/build_test_jshooks.sh

```diff
@@ -1,10 +1,34 @@
-#!/bin/bash
-CURRENT=$(cd $(dirname $0);pwd)
+#!/bin/bash -u
+
+# Generate the SetJSHook_wasm.h file from the SetJSHook_test.cpp file.
+# 
+# Prerequisites:
+# - quickjs:
+#   $git clone https://github.com/RichardAH/quickjslite.git
+#   $cd quickjslite
+#   $make
+#   copy `qjsc` to the src/test/app of the repo
+#
+# - clang-format:
+#   Ubuntu: $sudo apt-get install clang-format
+#   macOS: $brew install clang-format
```

Comment:
brew install r-lib/taps/clang-format@10    

To match the CI clang-format I had to use that. 19 wasn't working in every case. 

related to: 
https://github.com/Xahau/xahaud/pull/449/files#r1972938606